pkg/ocibundle/sif/bundle.go

```diff
@@ -59,8 +61,21 @@ func (s *sifBundle) Create(ociConfig *specs.Spec) error {
 	offset := uint64(part.Fileoff)
 	size := uint64(part.Filelen)
 
+	// TODO: embed OCI image config in SIF directly and generate
+	// runtime configuration from this one for environment variables
+	// and command/entrypoint
+	config := ociConfig
```

Comment:
What's the point of doing this here?

Reply:
This is a temporary workaround to run image without edit `config.json` to run corresponding service (eg: https://asciinema.org/a/80cAZMCcs8vFYTUSH1DVXGgHQ)